var/spack/repos/builtin/packages/kahip/package.py

```diff
@@ -0,0 +1,82 @@
+##############################################################################
+# Copyright (c) 2017 Christian Schulz
```

Comment:
The license should be LGPL, since this is what spack ships with and what is needed for overall license compatibility. The copyright notice is independent of this though. As I understand it, the copyright is what buttresses the license conditions. In the Anglo-saxon system copyright is used as a blanket term for author rights and usage rights. In other jurisdictions (eg, Germany), the author has control of [recognition and designation](http://www.gesetze-im-internet.de/englisch_urhg/englisch_urhg.html#p0057), but this copyright can only be [inherited, not transferred](http://www.gesetze-im-internet.de/englisch_urhg/englisch_urhg.html#p0141).
People could try to designate LLNL for the copyright, but the authorship is always a natural person (never an institution). This would thus merely be a bit of hand-waving for German courts, but not actually constitute a copyright transferal either.